miprometheus/utils/sampler_factory.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) IBM Corporation 2018
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+sampler_factory.py: Factory building samplers used by PyTorch's DataLoaders.
+
+"""
+__author__ = "Tomasz Kornuta"
+
+import os
+import logging
+import torch.utils.data.sampler
```

Comment:
LGTM warned that `inspect` was a useless import, so I removed it :wink:

Reply:
perfect, thx!